tests/integration/test_api_resources.py

```diff
@@ -52,17 +52,15 @@ def test_successful_upload_when_file_is_opened_in_binary_mode(self):
 
 
 class FileUploadFromUrlTest(RestAPITestCase):
+    image_url = 'https://github.githubassets.com/images/modules/logos_page/Octocat.png'
```

Comment:
We use this image in our integration tests and it been stably available for years: `https://www.google.ru/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png`
Bit github is all good :)